tests/trading/test_orders.py

```diff
@@ -552,576 +597,600 @@ def runtest(testCase):
             "sender": tester.a0,
             "outcome": 1,
             "fxpMoneyEscrowed": fix('0.61'),
             "fxpSharesEscrowed": 0,
-            "betterOrderID": 0,
-            "worseOrderID": 0,
+            "betterOrderID": ZEROED_ORDER_ID,
+            "worseOrderID": ZEROED_ORDER_ID,
             "tradeGroupID": 0
         }, {
             "orderID": 20,
-            "type": 2,
+            "hashedOrderID": hashedOrderID20,
+            "type": ASK,
             "fxpAmount": fix('1'),
             "fxpPrice": fix('0.59'),
             "sender": tester.a0,
             "outcome": 1,
             "fxpMoneyEscrowed": fix('0.59'),
             "fxpSharesEscrowed": 0,
-            "betterOrderID": 0,
-            "worseOrderID": 0,
+            "betterOrderID": ZEROED_ORDER_ID,
+            "worseOrderID": ZEROED_ORDER_ID,
             "tradeGroupID": 0
         }],
         "expected": {
             "bestOrder": {
-                "bid": 17,
-                "ask": 20
+                "bid": hashedOrderID17,
+                "ask": hashedOrderID20
             },
             "worstOrder": {
-                "bid": 16,
-                "ask": 19
+                "bid": hashedOrderID16,
+                "ask": hashedOrderID19
             },
             "orders": [{
-                "betterOrderID": 17,
-                "worseOrderID": 16
+                "betterOrderID": hashedOrderID17,
+                "worseOrderID": hashedOrderID16
             }, {
-                "betterOrderID": 15,
-                "worseOrderID": 0
+                "betterOrderID": hashedOrderID15,
+                "worseOrderID": ZEROED_ORDER_ID
             }, {
-                "betterOrderID": 0,
-                "worseOrderID": 15
+                "betterOrderID": ZEROED_ORDER_ID,
+                "worseOrderID": hashedOrderID15
             }, {
-                "betterOrderID": 20,
-                "worseOrderID": 19
+                "betterOrderID": hashedOrderID20,
+                "worseOrderID": hashedOrderID19
             }, {
-                "betterOrderID": 18,
-                "worseOrderID": 0
+                "betterOrderID": hashedOrderID18,
+                "worseOrderID": ZEROED_ORDER_ID
             }, {
-                "betterOrderID": 0,
-                "worseOrderID": 18
+                "betterOrderID": ZEROED_ORDER_ID,
+                "worseOrderID": hashedOrderID18
             }]
         }
     })
 
 def test_saveOrder(contractsFixture):
     market = contractsFixture.binaryMarket
-    orders = contractsFixture.contracts['orders']
-    ordersFetcher = contractsFixture.contracts['ordersFetcher']
-    order1 = 12345
-    order2 = 98765
+    orders = contractsFixture.contracts['Orders']
+    ordersFetcher = contractsFixture.contracts['OrdersFetcher']
+    hashedOrderID1 = orders.saveOrder(ZEROED_ORDER_ID, BID, market.address, fix('10'), fix('0.5'), tester.a1, NO, 0, fix('10'), ZEROED_ORDER_ID, ZEROED_ORDER_ID, 1)
```

Comment:
Style: I personally prefer a blank line between the test setup (like getting all of the contracts from the fixture) and when the test starts, and then when assertions happen.  Not all of our tests have clear separation like this, but this one could benefit from a blank line above this line I think.